app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,56 @@
+class TasksController < ApplicationController
+
+  def index
+    @tasks = Task.order(:completion_date)
+  end
+
+  def show
+    @task = Task.find(params[:id].to_i)
```

Comment:
Just a good practice to do something if you can't find the task, this applies to all of the actions like `edit`, `update`, `delete`, `mark_complete` etc.